bwc-test/src/test/resources/security/esnode.pem

```diff
@@ -1,28 +1,50 @@
-MDM0MzQ3WhcNMjgwNDE5MDM0MzQ3WjBeMRIwEAYKCZImiZPyLGQBGRYCZGUxDTAL
-BgNVBAcMBHRlc3QxDTALBgNVBAoMBG5vZGUxDTALBgNVBAsMBG5vZGUxGzAZBgNV
-BAMMEm5vZGUtMC5leGFtcGxlLmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
-AQoCggEBAJa+f476vLB+AwK53biYByUwN+40D8jMIovGXm6wgT8+9Sbs899dDXgt
-9CE1Beo65oP1+JUz4c7UHMrCY3ePiDt4cidHVzEQ2g0YoVrQWv0RedS/yx/DKhs8
-Pw1O715oftP53p/2ijD5DifFv1eKfkhFH+lwny/vMSNxellpl6NxJTiJVnQ9HYOL
-gf2t971ITJHnAuuxUF48HcuNovW4rhtkXef8kaAN7cE3LU+A9T474ULNCKkEFPIl
-ZAKN3iJNFdVsxrTU+CUBHzk73Do1cCkEvJZ0ZFjp0Z3y8wLY/gqWGfGVyA9l2CUq
-eIZNf55PNPtGzOrvvONiui48vBKH1LsCAwEAAaOCAVkwggFVMIG8BgNVHSMEgbQw
-gbGAFJI1DOAPHitF9k0583tfouYSl0BzoYGVpIGSMIGPMRMwEQYKCZImiZPyLGQB
-GRYDY29tMRcwFQYKCZImiZPyLGQBGRYHZXhhbXBsZTEZMBcGA1UECgwQRXhhbXBs
-ZSBDb20gSW5jLjEhMB8GA1UECwwYRXhhbXBsZSBDb20gSW5jLiBSb290IENBMSEw
-HwYDVQQDDBhFeGFtcGxlIENvbSBJbmMuIFJvb3QgQ0GCAQEwHQYDVR0OBBYEFKyv
-78ZmFjVKM9g7pMConYH7FVBHMAwGA1UdEwEB/wQCMAAwDgYDVR0PAQH/BAQDAgXg
-MCAGA1UdJQEB/wQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjA1BgNVHREELjAsiAUq
-AwQFBYISbm9kZS0wLmV4YW1wbGUuY29tgglsb2NhbGhvc3SHBH8AAAEwDQYJKoZI
-hvcNAQELBQADggEBAIOKuyXsFfGv1hI/Lkpd/73QNqjqJdxQclX57GOMWNbOM5H0
-5/9AOIZ5JQsWULNKN77aHjLRr4owq2jGbpc/Z6kAd+eiatkcpnbtbGrhKpOtoEZy
-8KuslwkeixpzLDNISSbkeLpXz4xJI1ETMN/VG8ZZP1bjzlHziHHDu0JNZ6TnNzKr
-XzCGMCohFfem8vnKNnKUneMQMvXd3rzUaAgvtf7Hc2LTBlf4fZzZF1EkwdSXhaMA
-1lkfHiqOBxtgeDLxCHESZ2fqgVqsWX+t3qHQfivcPW6txtDyrFPRdJOGhiMGzT/t
-e/9kkAtQRgpTb3skYdIOOUOV0WGQ60kJlFhAzIs=
+MIIEITCCAwmgAwIBAgIUGPqmZdMIOqjgbQ0nRdJ2F/VUkPgwDQYJKoZIhvcNAQEL
+BQAwga4xEzARBgoJkiaJk/IsZAEZFgNjb20xFzAVBgoJkiaJk/IsZAEZFgdleGFt
+cGxlMRkwFwYDVQQKDBBFeGFtcGxlIENvbSBJbmMuMSEwHwYDVQQLDBhFeGFtcGxl
+IENvbSBJbmMuIFJvb3QgQ0ExQDA+BgNVBAMMN0V4YW1wbGUgQ29tIEluYy4gUm9v
+dCBDQSwgQ04gPSBFeGFtcGxlIENvbSBJbmMuIFJvb3QgQ0EwHhcNMjMwODEwMjEx
+NTA3WhcNMzMwODA3MjExNTA3WjBXMQswCQYDVQQGEwJkZTENMAsGA1UEBwwEdGVz
+dDENMAsGA1UECgwEbm9kZTENMAsGA1UECwwEbm9kZTEbMBkGA1UEAwwSbm9kZS0w
+LmV4YW1wbGUuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9bgr
+ZoCVt85b6OlTOnaRhx/754+98el4tfSNxc1TEqJOgXd2WlGdPY6stcc+fLlHOECX
+D1u/QnaozcxFkVJAsjQbYIttSxldBWOrVLXJuw4fNmNCaZPAz88OloFCA6n6dma+
+K4nPPGrGiEe3c46+zU5SCio3Avs7qNTaoGINBJWLTqH9sHcWf9e2pDkJiwCq3Q8L
+WHwYGwZGAzw7wGDWFTO/g4VobS244s7JHNtLtRJ+FIpFlsKvtZlkdg8rw6EbkbXp
+v/yWJiu/gWM11emk5uxK8PCgFqVC+Rex3IJhlhziDxDzX7JAdXoHSqE3Q0aLCjjl
+cR1QK+/n41o/OtdASQIDAQABo4GMMIGJMEcGA1UdEQRAMD6CEm5vZGUtMC5leGFt
+cGxlLmNvbYIJbG9jYWxob3N0hxAAAAAAAAAAAAAAAAAAAAABhwR/AAABiAUqAwQF
+BTAdBgNVHQ4EFgQUwEDN1zWWQ4ULvKofWCRi+a/uTt8wHwYDVR0jBBgwFoAU6bJR
+I4km9IzZNac3Yt+Jyn9KkskwDQYJKoZIhvcNAQELBQADggEBAJd/5+cVSmsL2Gx4
+Dq36U4XOHfUSIVJUuOuu19ZxQ5a9ObUfNX0gSLXbcWnLcdOCUgN4OutjxUw16FeL
+G9t2tT/dKbJZiCkF0QPorugNGeeMt0fExRQ3D0a8OisGdqYurcHxVDjViyWurh1B
+ETFU6Gm2E1Rya3zCrW5wEll2GSNe1Xdc+1ANh1mW8E8TbzX3vhY6gXu9o8uNCXLU
+N0JE0ggugkOBLSSP9MbFvQDWWAY7nMaQxuXwULyiXMSm2QQOOlNqcGMASWo7sWij
+ELDpYc5XPVplpQObdTkEFrooHe7oBl91WZ9oY9iu/H2Yx6HS/zJs6knZvwNxz3uY
+ZcvdYXw=
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't think the new certs with the IPv6 loopback address (::1) in the SAN are required here anymore since the publish and bind host are both set to 127.0.0.1. 

When there are multiple certs in a single PEM file its usually the entire chain of certs from the lowest level at the top (node certificate) to the highest level on the bottom (root certificate). In the cert reload tests, they have a chain include an intermediate signing certificate and all the certs are required in the node certificate file.